transport/rest/taker.go

```diff
@@ -169,12 +169,14 @@ func (h *Handler) handleQuote(w http.ResponseWriter, r *http.Request, isSwap boo
 		OutToken:  req.OutToken,
 		InAmount:  req.InAmount,
 		InToken:   req.InToken,
-		SwapId:    "",
+		//SwapId:    "",
 		Fragments: []Fragment{},
 	}
 
-	if isSwap {
+	// debug
+	//AbiElements
 
+	if isSwap {
```

Comment:
Let me see if I understand: Is this handler is used for both quote and swap? When `isSwap = false`, the quote info gets returned?